eahub/base/static/scripts/maps/heatmap.js

```diff
@@ -71,8 +72,7 @@ function createMap() {
   return map
 }
 
-function addMarkersWithLabels(locations, map) {
-
+function addMarkersWithLabels(locations, map, checkGDPR = false) {
```

Comment:
Wouldn't it make more sense to just get rid of `checkGDPR` entirely?

Reply:
yes. Branch updated